CONTRIBUTORS.json

```diff
@@ -294,4 +295,34 @@
             "C"
         ]
     }
+    {
```

Comment:
your data is already present in `Contributors.json` file

- remove new record ( duplicate ) one 
- add your new project contribution in previous project array of object.